README.md

```diff
@@ -1,2 +1,57 @@
+
+It contains too the Firefox driver that it is not fully tested as currently the provissioning app does not work with Firefox
+
+## Usage
+You need to set several environment variables in order to make this work, as it is intended to be used in a container / pod lately.
+
+This can be done using the script provided:
+```sh
+$ . ./set-env.sh
+```
+
+| Variable              | Value                            | Description                                                                  |
+|-----------------------|----------------------------------|----------------------------------------------------------------------------- |
+| PROV_APP_USER         | openshift                        | Provisioning app user name                                                   |
+| PROV_APP_PASSWORD     | openshift                        | Provisioning app password                                                    |
+| ATLASSIAN_USER        | openshift                        | Atlassian user name                                                          |
+| ATLASSIAN_PASSWORD    | openshift                        | Atlassian password                                                           |
+| JENKINS_USER          | developer                        | Jenkins user name                                                            |
+| JENKINS_PASSWORD      | any                              | Jenkins password                                                             |
+| OPENSHIFT_USER        | developer                        | Openshift user name                                                          |
+| OPENSHIFT_PASSWORD    | any                              | Openshift password                                                           |
+| PROV_APP_NAME         | openshift                        | Name of the deployment of the provisioning app                               |
+| JIRA_URL              | http://jira.odsbox.lan:8080/     | Url of Jira instance related with the prov app                               |
+| OPENSHIFT_PROJECT     | edpp                             | project identifier for prov app in the preliminary tests(jira tests)         |
+| OPENSHIFT_PUBLIC_HOST | ocp.odsbox.lan                   | host where we can locate the prov app                                        |
+| OPENSHIFT_CLUSTER     | https://ocp.odsbox.lan:8443/     | URL of the Openshift Cluster                                                 |
+| BITBUCKET_URL         | http://bitbucket.odsbox.lan:7990 | Url of Bitbucket instance                                                    |
+| SIMULATE              | false                            | Specify (true or false)  if we skip the creation of project, components, etc |
+
+The following commands will launch the tests with the individual browsers:
+
+```shell script
+# Headless tests, to use chrome without displaying the browser.
+$ ./gradlew chromeHeadlessTest --tests "org.ods.e2e.ODSSpec"
```

Comment:
@jorge-romero Can we please add a Makefile with a make target named `test` so we can call `make test`? This would mean that we need to specify a `gradlew` command as a default with potentially additional make targets for specific browser types.

Reply:
I already create a issue regarding the use of make to launch he process, and grab the configuration

Reply:
We'lll start to add the makefile to launch the tests and later we will implement the way of retrieving the configuration.